pkg/aggregator/aggregator.go

```diff
@@ -607,6 +628,14 @@ func (agg *BufferedAggregator) GetEvents() metrics.Events {
 	return events
 }
 
+// GetEventPlatformEvents grabs the event platform events from the queue and clears them.
+// Note that this works only if using the 'noop' event platform forwarder
+func (agg *BufferedAggregator) GetEventPlatformEvents() map[string][]*message.Message {
+	agg.mu.Lock()
+	defer agg.mu.Unlock()
```

Comment:
```suggestion
```

Seems not needed anymore since https://github.com/DataDog/datadog-agent/commit/84cc39e026a3e8b41d8b3916577405125779dce8  ?

Reply:
Yes good catch! Removed